web/templates/web/participant-email-preferences-remove-all.html

```diff
@@ -0,0 +1,19 @@
+{% extends "web/base.html" %}
+{% load i18n %}
+{% block title %}
+    {% trans "Email Preferences Remove All" %}
+{% endblock title %}
+{% block content %}
+    <div class="row">
+        <div class="col-4">{% include "accounts/_account-navigation.html" with current_page="email-preferences" %}</div>
+        <div class="col-8">
+            <div class="card">
+                <div class="card-body">
+                    You've been unsubscribed from all emails.  If this was a mistake click
+                    <a href="{% url 'web:email-preferences' %}">here</a>
+                    to update your email preferences.
+                </div>
```

Comment:
What if instead of having a _link_ to the email preferences form (which the left-hand navigation shows is the current page), we actually include the email preferences form here and insert the "You've been unsubscribed..." message above it? That way they get confirmation that they're unsubscribed from all, along with a reminder of the different email categories and a quick way to change them again.
(I guess one problem with doing it this way is that it would make things more complicated if we decide to remove the login requirement.)
![Screenshot 2024-02-26 at 5 04 00 PM](https://github.com/lookit/lookit-api/assets/9041788/79454d24-b802-4f06-8f32-5ccb327e2038)

Reply:
OH!  I like that.

Reply:
I see now that my suggestion here isn't possible because the email preferences page/form requires a log in. Nevermind!